erpnext/telephony/doctype/call_log/call_log.json

```diff
@@ -134,15 +138,34 @@
    "fieldname": "call_details_section",
    "fieldtype": "Section Break",
    "label": "Call Details"
+  },
+  {
+   "depends_on": "to",
+   "fieldname": "employee_call_directed_to",
+   "fieldtype": "Data",
+   "label": "Employee Call Directed To",
```

Comment:
```suggestion
   "label": "Call Received By",
```
?

Reply:
call may not always be 'received' by the agent so i felt 'directed to' would make more sense, anyway I'll change